extensions.c

```diff
@@ -761,11 +761,15 @@ hibaext_sanity_check(const struct hibaext *ext) {
 				size_t i;
 				int quoted = 0;
 				int dquoted = 0;
+				int escaped = 0;
 
 				if (negative_matching)
 					ret = HIBA_UNEXPECTED_KEY;
 				else for (i = 0; i < strlen(value); ++i) {
 					switch (value[i]) {
+					case '\\':
+						escaped = (escaped+1)%2;
```

Comment:
For toggling, it might be more idiomatic to use:
`escaped = !escaped`

Reply:
true, although this is how the toggling is done for quoted and dquoted, so I need to update it in all location for consistency.